solutions/USACO open19-cow-evolution.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: usaco-941
+title: USACO 2019 December Bronze - Cow Evolution
+author: Jesse Choe
+---
+
+Detailed analysis here: https://www.youtube.com/watch?v=4YTijr0BbrA&t=1045s
+
+[Official Analysis](http://www.usaco.org/current/data/sol_evolution_bronze_open19.html)
+
+<LanguageSection>
+<CPPSection>
+
+## C++ Implementation
```

Comment:
I don't think we need another C++ implementation since it's essentially the same as the one in the official analysis.